blocks/ads-block/features/ads/default.jsx

```diff
@@ -1,28 +1,34 @@
 /* eslint-disable react-hooks/rules-of-hooks */
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState } from 'react';
 import PropTypes from '@arc-fusion/prop-types';
 import styled from 'styled-components';
-import getProperties from 'fusion:properties';
+import Lazy from 'lazy-child';
 import { useFusionContext } from 'fusion:context';
 import adMap from './ad-mapping';
 import ArcAdminAd from './_children/ArcAdminAd';
-import ArcAdsInstance from './_children/ArcAdsInstance';
-import { getAdObject, setPageTargeting } from './ad-helper';
+import { getAdObject } from './ad-helper';
+import AdUnit from './_children/AdUnit';
 import './ads.scss';
 
 function generateInstanceId(componentId) {
   return `${componentId}-${Math.floor(Math.random() * 9007199254740991).toString(16)}`;
 }
 
+// eslint-disable-next-line arrow-body-style
+const setAdLabelVisibility = (props) => {
+  // istanbul ignore next
+  return props.displayAdLabel ? '' : 'display: none';
+};
+
 const StyledAdUnit = styled.div`
-  .arcad > div[id^='google_ads_iframe']:not(:empty):before {
+  .arcad > [id^='google_ads_iframe']:not(:empty)::before {
     content: "${(props) => props.adLabel}";
```

Comment:
is stylelint drunk? is this not a string?

Reply:
Haha I think it's complaining that it's using double-quotes instead of single-quotes 😆  I thought the `content` needed double-quotes but I'll see if single-quotes are valid syntax for styled component CSS.

Reply:
Yep single quotes seem to work fine! Linting error is fixed now 👍